packages/package_info/example/integration_test/package_info_test.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:package_info/package_info.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  const MethodChannel channel =
+      MethodChannel('plugins.flutter.io/package_info');
+  List<MethodCall> log;
+
+  channel.setMockMethodCallHandler((MethodCall methodCall) async {
+    log.add(methodCall);
+    switch (methodCall.method) {
+      case 'getAll':
+        return <String, dynamic>{
+          'appName': 'package_info_example',
+          'buildNumber': '1',
+          'packageName': 'io.flutter.plugins.packageinfoexample',
+          'version': '1.0',
+        };
+      default:
+        assert(false);
+        return null;
+    }
+  });
+
+  setUp(() {
+    log = <MethodCall>[];
+  });
+
+  test('fromPlatform', () async {
+    final PackageInfo info = await PackageInfo.fromPlatform();
+    expect(info.appName, 'package_info_example');
+    expect(info.buildNumber, '1');
+    expect(info.packageName, 'io.flutter.plugins.packageinfoexample');
+    expect(info.version, '1.0');
+    expect(
+      log,
+      <Matcher>[
+        isMethodCall('getAll', arguments: null),
+      ],
+    );
+  });
+}
```

Comment:
You may replace this file with [this](https://github.com/swift-kim/plugins/commit/c07b89dd545bd29fa016e512898c7acc01b9fb16) (a new derivative of [package_info_test.dart](https://github.com/flutter/plugins/blob/master/packages/package_info/example/integration_test/package_info_test.dart)) which better evaluates the functionality of this plugin on Tizen devices. You will also have to update `example/lib/main.dart` as described in the commit to make the tests pass on devices with tiny screens (wearables).

Reply:
done.